go/service/prove.go

```diff
@@ -83,6 +87,8 @@ func (ph *ProveHandler) StartProof(ctx context.Context, arg keybase1.StartProofA
 
 // Prove handles the `keybase.1.checkProof` RPC.
 func (ph *ProveHandler) CheckProof(ctx context.Context, arg keybase1.CheckProofArg) (res keybase1.CheckProofStatus, err error) {
+	ctx = libkb.WithLogTag(ctx, "PV")
+	defer ph.G().CTraceTimed(ctx, fmt.Sprintf("CheckProof(%v)", arg.SigID), func() error { return err })()
```

Comment:
prefer `CheckProof: SigID: %v` in these fn traces, makes log parsing easier since you know what the arg is without having to go back to this line